app/views/govuk_publishing_components/components/_checkbox.html.erb

```diff
@@ -16,7 +16,8 @@
     aria: {
       describedby: item[:hint].present? ? "#{id}-item-hint" : nil,
       controls: item[:conditional].present? ? "#{id}-conditional-#{index}" : nil
-  } %>
+    }
+  %>
```

Comment:
This was a more impressive change before someone fixed it ahead of me and I had to rebase.